.github/workflows/interface-unit-tests.yml

```diff
@@ -496,6 +496,10 @@ jobs:
         config:
           - device: default.qubit.legacy
             shots: None
+          - device: default.qubit
+            shots: None
+          - device: default.qubit
+            shots: 10000
```

Comment:
probably a task for the future, but we might want to try to support testing with partitioned shots. just tried locally and it's non-trivial effort